### DISCUSSION TITLE
[Feature] Possibility to get city where an image was taken "boundaries-based"

### DISCUSSION THREAD

**Discussion by kenchrcum** [immich-app/immich#11487] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I just learned from reading the issue https://github.com/immich-app/immich/issues/9674 , that "The geolocation is not strictly boundaries-based, but rather it applies the closest point in the database[...]". 
Is there any possibility to have an option to enable the city where the image was taken "strictly boundaries-based"?
I find it rather annoying to see a photo tagged with a neighboring city, although the picture was not taken there.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by zackpollard** [comment]
We don't currently store boundaries data in the database, so this is not possible. The geo location works exclusively based on centre points of places and the closest place to the point t the photo was taken. I am currently in the middle of working on adding boundary based geo location, but it is unlikely to be the default due to the space requirements in the database (upwards of 6GB for the table). I'm going to close this issue but keep an eye on the release notes as it'll probably drop in one of the next few upcoming releases 🙂

**Reply by kenchrcum** [reply]
Thanks for the information! Looking forward to see it implemented :smile:
